index.html

```diff
@@ -10,6 +10,21 @@
   <link rel="stylesheet" href="application.css" type="text/css">
 </head>
 <body>
-  <h1 class="page-title">Front-End Tools</h1>
+  <div class="container">
+    <h1 class="text-big page-title">Front-End Tools</h1>
+    <div class="tech-section">
```

Comment:
This div is for this technology or for every technology in front end tools?

Reply:
the div tech-section is to put all tech-container (one for each tech)